packages/api/tests/serial/es/test-db-indexer.js

```diff
@@ -288,8 +285,32 @@ test.serial('create, update and delete an execution in dynamodb and es', async (
   records = await getDyanmoDBStreamRecords(process.env.ExecutionsTable);
 
   // fake the lambda trigger
-  await handler(records, {}, noop);
+  await handler(records);
 
   indexedRecord = await recordIndex.get(fakeRecord.arn);
   t.is(indexedRecord.detail, 'Record not found');
 });
+
+test.serial.only('The db-indexer does not throw an exception when execution fails', async (t) => {
```

Comment:
I think you want to remove the `.only` here?

Reply:
Fixed.